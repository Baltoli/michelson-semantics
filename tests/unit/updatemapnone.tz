$Instructions { UPDATE } ; 
$Stack ListItem(1) ListItem(None) ListItem(#Map(1 |-> 3 5 |-> 2, int, int));
