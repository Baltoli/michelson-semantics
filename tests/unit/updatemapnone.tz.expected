<generatedTop>
  <k>
    .
  </k>
  <stack>
    #Map ( 5 |-> 2 , int .TypeAnnotationList .FieldAnnotationList , int .TypeAnnotationList .FieldAnnotationList )
  </stack>
  <paramtype>
    #NotSet
  </paramtype>
  <storagetype>
    #NotSet
  </storagetype>
  <balance>
    #Mutez ( 0 )
  </balance>
  <amount>
    #Mutez ( 0 )
  </amount>
  <now>
    #Timestamp ( -1 )
  </now>
  <myaddr>
    #Address ( "InvalidMyAddr" )
  </myaddr>
  <knownaddrs>
    .Map
  </knownaddrs>
  <sourceaddr>
    #Address ( "InvalidSourceAddr" )
  </sourceaddr>
  <senderaddr>
    #Address ( "InvalidSenderAddr" )
  </senderaddr>
  <chainid>
    #ChainId ( 0 )
  </chainid>
</generatedTop>
